minio/resource_minio_s3_object.go

```diff
@@ -94,12 +94,17 @@ func minioPutObject(ctx context.Context, d *schema.ResourceData, meta interface{
 		return NewResourceError("putting object failed", d.Id(), errors.New("one of source / content / content_base64 is not set"))
 	}
 
+	options := minio.PutObjectOptions {}
```

Comment:
Thanks for this fix, @linyinfeng!
Code looks good, and we just need to make it pass the format-linter to get it ready to merge!

```suggestion
	options := minio.PutObjectOptions{}
```

Reply:
Thank you! I have run gofmt applied this change. Now it should pass the format check.